package.json

```diff
@@ -113,7 +113,7 @@
     "build:dev": "npm run clean && NODE_ENV=\"development\" webpack --progress --colors",
     "build:prod": "npm run test && npm run build",
     "build:deploy": "NODE_ENV=\"deploy\" webpack --progress --colors --target=web",
-    "watch": "NODE_ENV=\"development\" webpack --progress --colors --watch --cache=false",
+    "watch": "NODE_ENV=\"deploy\" webpack --progress --colors --watch --cache=false",
```

Comment:
Yes I did @mogoodrich , because using just `development` I don't know what the problem was but the application was not been updated in the reference app when it's on watch mode. So I found out that deploying to the `appdata/owa` folder work as it should do. I think someone else experienced the same issue, @Efosaok I think.

Reply:
Cool... I defer to you what it should be, but just want to double-check you meant to do it... :)

Reply:
@mogoodrich I have made adequate modifications., please review.